PhysicsTools/NanoAOD/python/particlelevel_cff.py

```diff
@@ -91,9 +91,13 @@
 
 rivetMetTable = simpleSingletonCandidateFlatTableProducer.clone(
     src = cms.InputTag("particleLevel:mets"),
-    name = cms.string("FiducialMET"),
+    name = cms.string("PFMET"),
     doc = cms.string("MET from Rivet-based ParticleLevelProducer in fiducial volume abs(eta)<5"),
-    variables = cms.PSet(PTVars),
+    extension = cms.bool(True),
+    variables = cms.PSet(
+       fiducialGenPt  = Var("pt",  float, precision=10),
+       fiducialGenPhi = Var("phi", float, precision=10),
```

Comment:
Let's take the opportunity to make the FiducialMET their own object with Pt+Phi, so that it allows for automatic 4-vector conversion in coffea etc.